data/seeds/02-users.js

```diff
@@ -1,13 +1,13 @@
 exports.seed = function(knex, Promise) {
   return knex('users').insert([
-    { name: 'Frodo Baggings' }, // 1
+    { name: 'Frodo Baggins' }, // 1
```

Comment:
so few people fix this... thank you. shouldv'e been fixed in the repo ages ago :cry: